MCprep_addon/tracking.py

```diff
@@ -260,10 +279,13 @@ def initialize(self, appurl, version, language=None, blender_version=None):
 		self._blender_version = str(blender_version)
 		self.language = language
 
-		self._tracker_idbackup = os.path.join(os.path.dirname(__file__),
-							os.pardir,self._addon+"_trackerid.json")
-		self._tracker_json = os.path.join(os.path.dirname(__file__),
-							self._addon+"_tracker.json")
+		self._tracker_idbackup = os.path.join(
+			os.path.dirname(__file__),
+			os.pardir,
+			self._addon + "_trackerid.json")
+		self._tracker_json = os.path.join(
+			os.path.dirname(__file__),
+			self._addon + "_tracker.json")
```

Comment:
I think it'll be worthwhile to see how this would look with `pathlib`, since `pathlib` tends to be clearer

Reply:
idk I'm still a bit torn on pathlib always being better. It means we're more likely to stick with long lines, whereas right now I sort of like the multi line input arguments as clearly each segment of the path.

is this how you'd write that first line? (noting this goes over 80 charcters)

```
		self._tracker_idbackup = Path(
			os.path.dirname(__file__) / os.pardir / self._addon + "_trackerid.json")
```

In retrospect, the lines where we are using Pathlib are the ones most exceeding line lengths now in files like conf.py (even if we push to a newline after the `(` in `Path(`, still exceeds line length for the definition of `self.json_path_update`)

I'll probably stick with this and merge this in the meantime, but feel free to comment after merging if I'm overlooking something.